mintpy/utils/isce_utils.py

```diff
@@ -917,3 +917,123 @@ def get_sensing_datetime_list(proj_dir, date_list=None):
 
     return sensingMid, sensingStart, sensingStop
 
+############################## Standard Processing ###########################################
+
+def gaussian_kernel(Sx, Sy, sig_x, sig_y):
+    '''
+    Generate a guassian kernal (with all elements sum to 1)
+    inputs: Sx, Sy - dimensions of kernal
+            sig_x, sig_y: standard deviation of the guassian distribution
+    '''
+    if np.mod(Sx,2) == 0:
+        Sx = Sx + 1
+
+    if np.mod(Sy,2) ==0:
+            Sy = Sy + 1
+
+    x,y = np.meshgrid(np.arange(Sx),np.arange(Sy))
+    x = x + 1
+    y = y + 1
+    x0 = (Sx+1)/2
+    y0 = (Sy+1)/2
+    fx = ((x-x0)**2.)/(2.*sig_x**2.)
+    fy = ((y-y0)**2.)/(2.*sig_y**2.)
+    k = np.exp(-1.0*(fx+fy))
+    a = 1./np.sum(k)
+    k = a*k
+    return k
+
+def convolve(data, kernel):
+    '''
+    return a convolved (filtered) complex image
+    inputs: data - complex array
+            kernel - convolution kernel
+    '''
+    from scipy import ndimage
```

Comment:
Could we move this to the top of the script?

Reply:
Moved to top of isce_utils.py